recipes/metabolisHMM/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Decompress the HMM database
+tar zxvf metabolisHMM_markers_v1.9.tgz
+
+# Install python libraries
+python3 -m pip install .
```

Comment:
Please add the `--no-deps` parameter (best practice is `python -m pip install . --no-deps -vv`).